doc/changelog_user_guide.md

```diff
@@ -0,0 +1,396 @@
+| `pull_request_change_template`        | `string`   | A template for each pull-request made by contributors of the repository. The default is `- {{ pr.title }} (PR #{{ pr.number }}) by @{{ pr.author }}`. The difference with `owners_pull_request_change_template` is that it includes the mention of the contributor.
+| `include_commits`                     | `bool`     | Defines whether to include commits - not present in pull-requests - in the changelog. Default is `true`.
+| `defaults`                            | `bool`     | Defines whether to add all pre-configured default categories and auto-labelers. Default is `true`. See configuration following sections.
+| `owners`                              | `string[]` | The list of owners to consider in the changelog. By default, the owner defined in the `[github]` section is added automatically if `defaults = true`.
+| `include.labels`                      | `string[]` | The list of labels that are _required_ to be present on a pull-request to be listed in the changelog. By default this list is empty.
+| `include.contributors`                | `string[]` | The list of contributors that are _required_ for the author of a pull-request to be listed in the changelog. By default this list is empty.
+| `exclude.labels`                      | `string[]` | The list of labels that if present on a pull-request will exclude the pull-request from being listed in the changelog. By default and if `defaults = true`, this list contains `skip-release-notes`.
+| `exclude.contributors`                | `string[]` | The list of contributors that if present on a pull-request will exclude the pull-request from being listed in the changelog.
+| `body_template`                       | `string`   | A template of the body of the release notes. The default is defined below.
+| `template_properties`                 | `map<string, object>`  | Allow to define user properties that are made accessible in all templates through the `properties` variable. See usage in following sections.
+| `[[changelog.autolabeler]]`           | `autolabeler` | The list of autolabelers configured. This is described in more details in the following sections.
+| `[[changelog.category]]`              | `category`    | The list of categories configured. This is described in more details in the following sections.
+
+The default of the `body_template` is the following [Scriban](https://github.com/scriban/scriban) template with `markdown`:
+
+```md
+# Changes
+
+{{ changes }}
+
+**Full Changelog**: {{ url_full_changelog_compare_changes }}
+```
+### 1.3 Categories
+
+Categories allow to define how the changes are organized within your release notes. By default if `defaults = true` is kept `true`, the following categories will be define.
```

Comment:
Probably should be something like:
- If `defaults` is kept **to** `true`, the following categories will be define**d**

Or:

- If `defaults = true` is kept, the following categories will be define**d**